blog/2024-03-13-magazine-008.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Inflight Magazine no. 8
+description: Updates from the winglang project
+authors:
+  - eladb
+  - revitalb
+tags:
+  - cloud-oriented
+  - programming
+  - winglang
+  - changelog
+  - newsletter
+hide_table_of_contents: true
+---
+
+> The 8th issue of the Wing Inflight Magazine.
+> <!--truncate-->
+
+Hello Wingnuts!
+
+We are back with another issue of the **Wing Inflight Magazine** and excited to share some updates about [Winglang](https://winglang.io), **an open source-programming language for the cloud**.
+
+Wing is shipped with a **powerful cloud simulator** which allows developers to write and test complete cloud application without having to deploy anything to the cloud.
+
+The language combines cloud infrastructure ***and*** runtime into a single programming model and **compiles to Terraform/CloudFormation and JavaScript** that are ready to deploy to AWS, GCP and Azure (see [support matrix](https://www.winglang.io/docs/standard-library/compatibility-matrix)).
+
+We are designing Wing to be **familiar and friendly** for developers who come from modern object-oriented background, so it will take you [5 minutes](https://github.com/winglang/workshop-react/blob/main/cheatsheet.md) to learn.
+
+Check out our [getting started tutorial](https://www.winglang.io/docs/) or hit the [Wing Playground](https://www.winglang.io/play/) for an online experience.
+
+### In today's issue
+
+- 🚀 [Deploying AWS resources and APIs inside a private VPCs](#w1)
+- 🎨 [A fresh look for the Wing Console map view](#w2)
+- 🤩 [New syntax for creating sets](#w3)
+- 🔥 [Mutually referential type declarations](#w4)
+- 🛟 [The bang `!` operator](#w5)
+- 🏋️ [Ability to lift collection of inflight closures](#w6)
+- 🗄️ [`fs.appendFile()`](#w7)
+- ⚙ [Platform Parameters and AWS VPC configuration](#w8)
+- 🛌 [Ergonomic improvements for `cloud.Function` and `cloud.Api`](#w9)
+- 🎥 [Goodies from the Wingly Update](#w10)
+- 👯‍♀️ [Community Events](#w11)
+
+> NOTE: I spent like two hours trying to figure out the right emojis here so please take a minute to appreciate them :-)
+
+### Deploying AWS resources and APIs inside a private VPCs
+<a id="w1"></a>
+
+![](./2024-03-13-private-api.png)
+
+Users tell us they are looking for ways to deploy API-based services inside their VPCs. We've spent some time learning about [the challenges people have](https://www.winglang.io/blog/2024/02/15/private-api-gateways) and figured we can offer a awesome experience in Wing for this use case.
+
+We published this [blog post](https://www.winglang.io/blog/2024/02/09/private-api-gateway-aws) which describes how to configure the `tf-aws` platform to deploy your application's AWS resources and APIs inside a VPC, with support for creating a whole new VPC or deploying into a VPC that's already setup in your environment.
```

Comment:
Do you mind changing the blog post to something like guide/tutorial?

Reply:
done